.github/workflows/rust.yml

```diff
@@ -8,7 +8,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 1
+        ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Maybe we should rebase v0.10 and then run the test again.
Seems like this PR is conflicting with https://github.com/nervosnetwork/godwoken/commit/9a0602b47668cf5658df03f46aa75298081d2728.

Reply:
```acionts/checkout@v3``` somehow try to merge this pr with develop branch, not 0.10 branch.

Reply:
This branch merged c33e49fe59b79e7d0877744dcd309dc3a07fc14a (feat-fast-withdrawal-to-v1) into e597b47bbe3cad9251a9b4c53925c3600ffa42df (v0.10).
The [test](https://github.com/Flouse/godwoken/runs/6172402897?check_suite_focus=true) was failed.


```js
commit 4252c056887cf35a7f5f862d76a8e7244de1ee06 (HEAD)
Merge: e597b47b c33e49fe
Author: zeroqn <zero.forest@outlook.com>
Date:   Wed Apr 20 12:55:49 2022 +0000

    Merge c33e49fe59b79e7d0877744dcd309dc3a07fc14a (feat-fast-withdrawal-to-v1) into e597b47bbe3cad9251a9b4c53925c3600ffa42df (v0.10)
```

Reply:
Please look at the error message
```
   --> crates/tools/src/withdraw_to_v1.rs:166:27
    |
166 |     let withdrawal_hash = godwoken_rpc_client.submit_withdrawal_request(bytes)?;
    |                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the `?` operator cannot be applied to type `impl std::future::Future`
```
It complain about ```Future```, but `godwoken_rpc_client.submit_withdrawal_request` in v0 isn't ```async fn```.
It's ```async fn``` in v1.

Please clone the pr and run test local.

Reply:
My mistake.
Didn't notice new commits...
I'll rebase code.